Dockerfile

```diff
@@ -4,11 +4,11 @@ FROM ekino/java7
 MAINTAINER fronton@ekino.com
 
 ENV WM_PACKAGE wiremock
-ENV WM_VERSION 1.55
+ENV WM_VERSION 2.1.11
 
 RUN mkdir /$WM_PACKAGE
 WORKDIR /$WM_PACKAGE
-RUN curl -sSL -o $WM_PACKAGE.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock/$WM_VERSION/$WM_PACKAGE-$WM_VERSION-standalone.jar
+RUN curl -sSL -o $WM_PACKAGE.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock/$WM_VERSION/$WM_PACKAGE-$WM_VERSION.jar
```

Comment:
The correct repo is `http://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.1.9/`